lib/SILGen/SILGenFunction.cpp

```diff
@@ -441,30 +441,25 @@ void SILGenFunction::emitArtificialTopLevel(ClassDecl *mainClass) {
     // we're getting away with it because the types are guaranteed to already
     // be imported.
     ASTContext &ctx = getASTContext();
-    ModuleDecl *UIKit = ctx.getLoadedModule(ctx.getIdentifier("UIKit"));
+    
+    std::pair<Identifier, SourceLoc> UIKitName =
+      {ctx.getIdentifier("UIKit"), SourceLoc()};
+    
+    ModuleDecl *UIKit = ctx
+      .getClangModuleLoader()
+      ->loadModule(SourceLoc(), UIKitName);
```

Comment:
This will load the module if it hasn't been loaded yet. Can you at least assert that `getLoadedModule` works before dropping down to the Clang level?

Reply:
I can add an assertion. We always load the UIKit module when the `@UIApplicationMain` attribute is checked in Sema, so that shouldn't be an issue in practice.